pedicel.gemspec

```diff
@@ -14,14 +14,14 @@ Gem::Specification.new do |s|
 
   s.files = Dir.glob("lib/**/*.rb")
 
-  s.add_runtime_dependency 'dry-validation', '1.9'
+  s.add_runtime_dependency 'dry-validation', '~> 1.9'
   s.add_runtime_dependency 'dry-schema', '~> 1.9'
   s.add_runtime_dependency 'dry-logic', '~> 1.0'
 
-  s.required_ruby_version = '~> 2.7.4'
+  s.required_ruby_version = '>= 2.7.4'
 
+  # s.add_development_dependency 'pedicel-pay'
```

Comment:
Temporarily commented. 

Remove the comment and add version requirement after [this PR](https://github.com/clearhaus/pedicel-pay/pull/16) is accepted.

Reply:
```suggestion
  s.add_development_dependency 'pedicel-pay'
```